lib/install/management/appliance.go

```diff
@@ -393,6 +383,12 @@ func (d *Dispatcher) findApplianceByID(conf *config.VirtualContainerHostConfigSp
 		return nil, err
 	}
 	vmm = vm.NewVirtualMachine(d.op, d.session, ovm.Reference())
+
+	element, err := d.session.Finder.Element(d.op, vmm.Reference())
```

Comment:
@cgtexmex I remember adding this since the resulting object did not have the inventory path after the type cast. Did you discover a better way to handle this? This code path looked a little worrisome to me when I wrote it.

Reply:
There's a _better_ way (or possibly just a different way), but won't be included now...requires a significant rework on how we acquire the appliance in `vic-machine`